### PR TITLE
Make Constraints do work for subviews

### PR DESCRIPTION
**Pull request by TingtingAn**
set TranslatesAutoresizingMaskIntoConstraints to false for all subviews.

### CODE REVIEW

**Comment by TingtingAn** [comment]
https://github.com/FluentLayout/Cirrious.FluentLayout/issues/43

File: Cirrious.FluentLayout/FluentLayoutExtensions.cs [hunk 1]
```diff
@@ -18,8 +18,8 @@ public static void SubviewsDoNotTranslateAutoresizingMaskIntoConstraints(this UI
             foreach (var subview in view.Subviews)
             {
                 subview.TranslatesAutoresizingMaskIntoConstraints = false;
-		
-	        if (subview.Subviews != null && subview.Subviews.Count() > 0)
+
+                if ((subview as UIControl) == null && subview.Subviews != null && subview.Subviews.Count() > 0)
```
Comment (Cheesebaron) [inline Cirrious.FluentLayout/FluentLayoutExtensions.cs]: Why do you cast to UIControl? You can't put anything else than subclasses of UIView into the subview collection and far from all controls will be UIControl...

Comment (TingtingAn) [inline Cirrious.FluentLayout/FluentLayoutExtensions.cs]: all system controls, that have some subviews, have a common superclass (UIControl).

eg:  `UIButton`, `UISwitch`, `UISegmentedControl`, `UISlider` and so on.

I just find an Constraint issue with `UISwitch` if without `(subview as UIControl) == null`

Comment (Cheesebaron) [inline Cirrious.FluentLayout/FluentLayoutExtensions.cs]: However, that means if I create my own view that subclasses `UIView` and nest other `UIView` inside, this piece of code won't work.

Comment (TingtingAn) [inline Cirrious.FluentLayout/FluentLayoutExtensions.cs]: @Cheesebaron   oh, yes, you are right.  So ignore the changes, any suggestion for the subviews Constraints issue at here?

**Pull request closed without merging**